<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="mwStackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="timePage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="GraphsButton_2">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 132, 26);</string>
              </property>
              <property name="text">
               <string>Graphs</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PresentButton_2">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 132, 26);</string>
              </property>
              <property name="text">
               <string>Present</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="HistoryButton_2">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 132, 26);</string>
              </property>
              <property name="text">
               <string>History</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QStackedWidget" name="timeStackedWidget">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <widget class="QWidget" name="presentTimePage">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>1</number>
              </property>
              <item>
               <widget class="QListView" name="currentActivitiesListView">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
                <property name="resizeMode">
                 <enum>QListView::Adjust</enum>
                </property>
                <property name="gridSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="debugButton">
                <property name="text">
                 <string>GetData</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableView" name="activitiesTableView">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
                <attribute name="horizontalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="historyTimePage">
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>130</x>
                <y>190</y>
                <width>161</width>
                <height>91</height>
               </rect>
              </property>
              <property name="text">
               <string>history</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="graphsTimePage">
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>140</x>
                <y>0</y>
                <width>131</width>
                <height>111</height>
               </rect>
              </property>
              <property name="text">
               <string>graphs</string>
              </property>
             </widget>
             <widget class="QTableView" name="debugTableView">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>170</y>
                <width>256</width>
                <height>192</height>
               </rect>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="plansPage">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>180</y>
          <width>191</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string>plans</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="financePage">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>200</y>
          <width>161</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string>finance</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="TimeButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 132, 26);</string>
        </property>
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PlansButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 85, 255);</string>
        </property>
        <property name="text">
         <string>Plans</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FinancesButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 241, 85);
alternate-background-color: rgb(148, 140, 49);</string>
        </property>
        <property name="text">
         <string>Finances</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
